### PR TITLE
fix(i18n): Missed 'Replies' in en.php

### PR DESCRIPTION
**Pull request by RiverVanRain**

### CODE REVIEW

**Comment by juho-jaakkola** [comment]
You have used the correct commit message format, but you've accidentally used it in the title of the pull request instead of the commit message (which currently is "Missing 'Replies' in en.php").

You can edit the commit message of an existing PR like this: https://help.github.com/articles/changing-a-commit-message/

Also isn't this string related to discussions plugin instead of the groups plugin? If so, it should be added there and changed to ```discussions:replies```.

File: mod/groups/languages/en.php [hunk 1]
```diff
@@ -123,6 +123,7 @@
 	'admin:groups' => 'Groups',
 
 	'groups:latestdiscussion' => 'Latest discussion',
+	'group:replies' => 'Replies',
```
Comment (jdalsem) [inline mod/groups/languages/en.php]: shouldn't this be `groups:replies`? Where is this used?

Comment (juho-jaakkola) [inline mod/groups/languages/en.php]: See the last part of my earlier comment.

**Comment by ewinslow** [comment]
This has not gotten any attention from @RiverVanRain in 2 weeks and has several blocking issues. Please reopen if/when there is renewed interest.

**Pull request closed without merging**